File: Array 101/Height_Checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        int arr[heights.size()];
        for(int i = 0;i<heights.size();i++)
        {
            arr[i]=heights[i];
        }
        sort(heights.begin(),heights.end());
        int flag=0;
        for(int j=0;j<heights.size();j++)
        {
            if(arr[j]!=heights[j])
                flag++;
        }
        return flag;
    }
};

File: Array 101/Height_Checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        arr = []
        for i in range(len(heights)):
            arr.append(heights[i])
        heights.sort()
        flag = 0
        for i in range(len(heights)):
            if arr[i] != heights[i]:
                flag += 1
        return flag
        